conan/test_package/CMakeLists.txt

```diff
@@ -1,49 +1,70 @@
 cmake_minimum_required(VERSION 3.1)
 project(PackageTest CXX)
 
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
 find_package(userver REQUIRED)
 find_package (Python COMPONENTS Interpreter Development)
 
 add_library(${PROJECT_NAME}_objs OBJECT
     hello.hpp
     hello.cpp
 )
-target_link_libraries(${PROJECT_NAME}_objs PUBLIC userver::core)
+target_link_libraries(${PROJECT_NAME}_objs PUBLIC userver::core userver::universal)
 
 add_executable(${PROJECT_NAME}_core test_core.cpp)
-target_link_libraries(${PROJECT_NAME}_core ${PROJECT_NAME}_objs userver::utest)
+target_link_libraries(${PROJECT_NAME}_core ${PROJECT_NAME}_objs)
 
-add_executable(${PROJECT_NAME}_utest test_utest.cpp)
-target_link_libraries(${PROJECT_NAME}_utest ${PROJECT_NAME}_objs userver::utest)
-add_google_tests(${PROJECT_NAME}_utest)
+add_executable(${PROJECT_NAME}_universal test_universal.cpp)
+target_link_libraries(${PROJECT_NAME}_universal ${PROJECT_NAME}_objs userver::core userver::universal)
```

Comment:
the universal should work event without the core, it is not dependent from core
so, remove the linkage to core